### PR TITLE
FWF499

### PR DESCRIPTION
**Pull request by kurianbenoy-aot**
image

- On updating date from a search type of Date to another variable, we are showing corresponding ?? symbol in UI and on entering value the user can search corresponding to that API

### CODE REVIEW

File: camunda-formio-tasklist-vue/src/components/search/TaskListSearch.vue [hunk 1]
```diff
@@ -279,11 +285,14 @@ export default class TaskListSearch extends Vue {
     }
     Vue.set(this.selectedSearchQueries, index, searchitem);
     this.operator[index] = this.selectedSearchQueries[index].compares[0];
-    if(this.selectedSearchQueries[index].type==='date'){
```
Comment (ann-aot) [inline camunda-formio-tasklist-vue/src/components/search/TaskListSearch.vue:282]: does compares array has at least one element?- will it throw any error?

**Review by ann-aot — APPROVED** [review]

**Comment by ann-aot** [comment]
pls pull the latest code from upstream and resolve the conflicts

File: camunda-formio-tasklist-vue/src/components/search/TaskListSearch.vue (hunk 1 above)
Comment (kurianbenoy-aot) [reply camunda-formio-tasklist-vue/src/components/search/TaskListSearch.vue:282]: Compares array will have all it's elements prefilled every time. Hence it won't throw any error during program execution

image

**Comment by kurianbenoy-aot** [comment]
Resolved the merge conflicts

**Comment by kurianbenoy-aot** [comment]
Merging #8

**Pull request merged**